### PR TITLE
Use new "saucelabs" NPM package to speak to Sauce API

### PR DESCRIPTION
**Pull request by christian-bromann**
**Is your feature request related to a problem? Please describe.**
As part of a product here at Sauce I've rewritten the [`saucelabs`](https://www.npmjs.com/package/saucelabs) NPM package.

**Describe the solution you'd like**
We should consume that package in the sauce service instead of building our own URL.

### CODE REVIEW

File: packages/webdriverio/src/scripts/isElementDisplayed.js [hunk 1]
```diff
@@ -180,9 +180,8 @@ export default function isElementDisplayed(element) {
         if (element.type === 'hidden')
             return false
         break
-
-    //case 'MAP':
-    // FIXME: Selenium has special handling for <map> elements. We don't do anything now.
+        // case 'MAP':
```
Comment (abjerstedt) [inline packages/webdriverio/src/scripts/isElementDisplayed.js]: Lint i assume?

Comment (christian-bromann) [reply packages/webdriverio/src/scripts/isElementDisplayed.js]: yes

**Comment by abjerstedt** [comment]
@christian-bromann Merge Conflicts

**Comment by christian-bromann** [comment]
fixed them

**Pull request merged**